### DISCUSSION TITLE
VPN routing through Android Wifi Hotspot

### DISCUSSION THREAD

**Discussion by Csantos88** [guardianproject/orbot-android#945] [Ideas]
Hi dev's.

If not implemented, there's a way to routing Orbot VPN mode through Android WIfi hotspot? 
I think there will be great function deep down the VPN do system level, if possible

**Comment by Csantos88** [comment]
*VPN **TO** system level, if possible

**Comment by n8fr8** [comment]
It isn't really possible unfortunately. Here is some info on the subject from the Proton VPN folks: https://protonvpn.com/support/share-vpn-connection-android-hotspot/

Now, within Orbot, there is an HTTP and SOCKS proxy that you can configure any device on your hotspot to use. That may be on way to do it, but it won't be automatic like you are hoping with the VPN.
